skore/src/skore/sklearn/_plot/data/table_report.py

```diff
@@ -0,0 +1,49 @@
+from skrub._reporting._html import to_html
+from skrub._reporting._summarize import summarize_dataframe
+
+from skore.sklearn._plot.base import Display
+
+
+class TableReportDisplay(Display):
```

Comment:
Actually, you don't inherit from a Python `Protocol` ;). It is only useful to do something like `isinstance(table_report, Display)` without forcing inheritance and just checking the availability of methods.

So here, you need to do:

```suggestion
class TableReportDisplay(StyleDisplayMixin, HelpDisplayMixin):
```

Reply:
Ok, got it, but note that `ConfusionMatrixDisplay` inherits from `Display`

Reply:
Yep I open some time ago this issue: https://github.com/probabl-ai/skore/issues/1757